rlclientlib/dedup.cc

```diff
@@ -315,6 +314,7 @@ class dedup_extensions : public logger::i_logger_extensions
 	}
 private:
 	dedup_state _dedup_state;
+  int _dummy_state;
```

Comment:
While I think this is a waste of space and we could get away with making `_dummy` static, I think this alternative is more sound and principled.